### ISSUE TITLE
How do i use 7zip

### ISSUE THREAD

**Issue by PatrickJ21** [boranblok/nntpPoster#80]
Hello how can i use 7zip with your software?? i am getting errors.

i have this in my config but i guess your program does not work for 7zip and only rar? any chance you can add support if that's the case ?

# Where to find rar on the system, leave empty if rar is accesible trough the path
RarLocation=C:\Program Files\7-Zip\7z.exe

# Extra parameters to add to the end of the rar command, careful with this as it can break stuff.
RarExtraParameters=

Log:

2023-04-13 18:06:16,870 DEBUG 5 ExternalProcessWrappers.ExternalProcessWrapperBase - Executing process: [C:\Program Files\7-Zip\7z.exe a -ep1  -m0 -r -v42240000b  "Q:\nntp\working\f724fd2daec74733953373f9482837f3_readyToPost\f724fd2daec74733953373f9482837f3" "Q:\nntp\working\default\f724fd2daec74733953373f9482837f3.mkv" ]
2023-04-13 18:06:16,893 WARN 12 ExternalProcessWrappers.ExternalProcessWrapperBase - Command Line Error:
2023-04-13 18:06:16,893 WARN 12 ExternalProcessWrappers.ExternalProcessWrapperBase - Unknown switch:
2023-04-13 18:06:16,893 WARN 12 ExternalProcessWrappers.ExternalProcessWrapperBase - -ep1
2023-04-13 18:06:16,897 INFO 5 nntpPoster.UsenetPoster - Deleting processed folder
2023-04-13 18:06:16,897 DEBUG 5 nntpPoster.NntpMessagePoster - Disposing Posting threads.
2023-04-13 18:06:17,062 ERROR 5 nntpAutoposter.AutoPoster - The upload failed to post. Retrying.
System.Exception: Process has exited with errors. Exit code: 7

**Comment by boranblok** [comment]
Yeah, I see the code is very .rar specific right now. I dont have much experience with how a typical 7zip archive is built on usenet. What would be the normal commandline for 7zip used in that case?
